package.json

```diff
@@ -35,6 +35,7 @@
     "ejs": "~2.3.4",
     "html-wiring": "~1.2.0",
     "mkdirp": "~0.5.1",
+    "underscore": "~1.8.3",
```

Comment:
Well, not really - in most projects I use exactly lodash. Here I went for underscore because that's what used to be included into yo. I will change it to lodash in the next push. 

> On 15 янв. 2016 г., at 7:32, Ilan Biala <notifications@github.com> wrote:
> 
> In package.json:
> 
> > @@ -35,6 +35,7 @@
> >      "ejs": "~2.3.4",
> >      "html-wiring": "~1.2.0",
> >      "mkdirp": "~0.5.1",
> > +    "underscore": "~1.8.3",
> Any reason to use underscore over lodash? Isn't lodash more performant and more maintained nowadays? I don't see this as an issue, but maybe just something to consider if changing it is just a matter of the dependency and using _.assign instead?
> 
> —
> Reply to this email directly or view it on GitHub.
>